datacube_stats/utils/create_source_files.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
```

Comment:
This file probably shouldn't live here, and contains lots of copy pasted code from elsewhere.